bitrepository-webclient/src/main/webapp/integrity-service.html

```diff
@@ -291,58 +324,91 @@ <h3 id="modalPagerLabel">Modal header</h3>
     }
 
     function updateStringCell(id, type, cellValue) {
-        var context = getCellContext(id, type);
-        var html = "<p>" + cellValue + "</p>";
+        let context = getBodyContext(id, type);
+        let html = "<p>" + cellValue + "</p>";
         $("#" + context.element).html(html);
     }
 
-    function updateIntCell(id, type, cellValue) {
-        var context = getCellContext(id, type);
-        if(cellValue == 0) {
-            var html = "<button class=\"btn btn-link\" disabled>" + formatInt(cellValue) + "</button>";
+    function updateIntCell(pillarID, type, cellValue) {
+        let context = getBodyContext(pillarID, type);
+
+        if (cellValue === 0) {
+            let html = `<button class="btn btn-link" disabled>${formatInt(cellValue)}</button>`;
             $("#" + context.element).html(html);
         } else {
-            var innerElement = context.element + "-a";
-            var html = "<a class=\"btn btn-link\" id=\"" + innerElement + "\">" + formatInt(cellValue) + "</a>";
+            let innerElement = context.element + "-a";
+            let html = `<a class="btn btn-link" id="${innerElement}">${formatInt(cellValue)}</a>`;
             $("#" + context.element).html(html);
-            $("#" + innerElement).click(showModalPager(id, context.member, context.title, context.url));
+            $("#" + innerElement).click(showModal(type, context.title, context.url, pillarID));
```

Comment:
I fixed this on my local branch that I'll push, but `context.member` got removed here, which left things not working.